.circleci/config.yml

```diff
@@ -221,10 +221,14 @@ workflows:
       - test-unit-jest-development:
           requires:
             - prep-deps
+      - test-integration:
```

Comment:
do you know what's the `test-integration-main` that appears in the [ci job](https://app.circleci.com/insights/github/MetaMask/metamask-extension/workflows/test_and_release/jobs?branch=cryptotavares%2Fpoc-integration-tests)? :thinking: 

I can't seem to find this change in this branch, but this job seems to only appear in this branch :thinking: 
![Screenshot from 2024-06-25 10-55-57](https://github.com/MetaMask/metamask-extension/assets/54408225/0cb17a41-1c95-4ebd-a184-067c42a65ca8)

Reply:
Initially we named it as `test-integration-main` but then thought that it would be better to just call it `test-integration` 😅

Reply:
ohhh that explains it! Thank you :laughing: